keyboards/packrat/keymaps/3Uc/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2021 drhigsby
```

Comment:
Keymap folder name should be lowercase.

Reply:
should be fixed with this commit https://github.com/qmk/qmk_firmware/pull/14574/commits/0ffd645d52fee2aebd36ae346db40fa538e39e9a